infra/modules/curi/mantarray_downloads/dns.tf

```diff
@@ -0,0 +1,71 @@
+resource "aws_route53_zone" "main" {
+  name = var.hosted_zone
+}
+
+resource "aws_route53_zone" "sub" {
+  name = "${var.subdomain}.${var.hosted_zone}"
+}
+
+
+resource "aws_route53_record" "ns_record" {
+  allow_overwrite = true
+  name            = var.hosted_zone
+  ttl             = 172800
+  type            = "NS"
+
+  zone_id = aws_route53_zone.main.zone_id
+  records = aws_route53_zone.main.name_servers
+}
+
+
+# resource "aws_route53_record" "sub_ns_record" {
+#   allow_overwrite = true
+#   name            = "${var.subdomain}.${var.hosted_zone}"
+#   ttl             = 172800
+#   type            = "NS"
+#   zone_id         = aws.route53_zone.sub.zone_id
+#   records         = aws_route53_zone.sub.name_servers
+# }
+
+
+resource "aws_route53_record" "a_record" {
+  zone_id = aws_route53_zone.main.zone_id
+  name    = var.hosted_zone
+  type    = "A"
+  ttl     = 3600
+
+  records = [
+    "216.239.32.21",
+    "216.239.34.21",
+    "216.239.36.21",
+    "216.239.38.21"
+  ]
+}
+
+
+resource "aws_route53_record" "mx_record" {
+  zone_id = aws_route53_zone.main.zone_id
+  name    = var.hosted_zone
+  type    = "MX"
+  ttl     = "300"
+
+  records = [
+    "1 ASPMX.L.GOOGLE.COM.",
+    "5 ALT1.ASPMX.L.GOOGLE.COM.",
+    "5 ALT2.ASPMX.L.GOOGLE.COM.",
+    "10 ALT3.ASPMX.L.GOOGLE.COM.",
+    "10 ALT4.ASPMX.L.GOOGLE.COM."
+  ]
+}
+
+
+resource "aws_route53_record" "txt_record" {
+  zone_id = aws_route53_zone.main.zone_id
+  name    = var.hosted_zone
+  type    = "TXT"
+  ttl     = 300
+
+  records = [
```

Comment:
double check if there's anything missing...I think there's a github verification tag in the DNS record also

Reply:
Is the github verifcation record specific to the curibio.com domain or can the new domains use the same verification?